website/docs/r/tg_connection_prefix_filter.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+subcategory: "Transit Gateway"
+layout: "ibm"
+page_title: "IBM : tg_connection_prefix_filter"
+description: |-
+  Manages IBM Transit Gateway Connection Prefix Filter.
+---
+
+# ibm_tg_connection_prefix_filter
+Create, update and delete for the transit gateways connection's prefix filter resource. For more information, about Transit Gateway connection prefix filters, see [adding and deleting prefix filters](https://cloud.ibm.com/docs/transit-gateway?topic=transit-gateway-adding-prefix-filters&interface=ui).
+
+## Example usage
+
+```terraform
+resource "ibm_tg_connection_prefix_filter" "test_tg_prefix_filter" {
+    gateway = ibm_tg_gateway.new_tg_gw.id
+    connection_id = ibm_tg_connection.test_ibm_tg_connection.connection_id
+    action = "permit"
+    prefix = "192.168.100.0/24"
+    le = 0
+    ge = 32
+}
+```
+
+## Argument reference
+Review the argument references that you can specify for your resource.
+
```

Comment:
missing other required arguments like gatewayid and connection id..
Please make sure all arguments and attributes defined in schema of resource/datasource are documented

Reply:
Good catch, I added the missing arguments and verified they are correct now